app/src/main/java/org/fossasia/openevent/app/data/cache/ObjectCache.java

```diff
@@ -0,0 +1,34 @@
+package org.fossasia.openevent.app.data.cache;
+
+
+import android.support.v4.util.LruCache;
+
+import org.fossasia.openevent.app.contract.model.CacheModel;
+
+public class ObjectCache implements CacheModel {
+
+    private static ObjectCache instance;
```

Comment:
@iamareebjamal we should [implement singleton with enum](http://stackoverflow.com/questions/26285520/implementing-singleton-with-an-enum-in-java). Current pattern  will not support unit testing.